scripts/schema-test-coverage.mjs

```diff
@@ -45,7 +46,15 @@ class TestCoveragePlugin {
     this.allLocations = [];
 
     for (const schemaLocation in context.ast) {
-      if (schemaLocation === "metaData") {
+      if (
+        schemaLocation === "metaData" ||
+        // Do not reqiure coverage of standard JSON Schema
+        schemaLocation.includes("json-schema.org") ||
+        // Do not require coverage of default $dynamicAnchor
+        // schemas, as they are not expected to be reached
+        // schemaLocation.includes("/schema/WORK-IN-PROGRESS#/$defs/schema/")
```

Comment:
Was this an idea that did not work, or is it a TODO for further investigation?

Reply:
Oh actually yeah last line was an older version with an error and I forgot to remove it when I figured out the trailing `/` was wrong.